Mempool node group page

softsimon opened the pull request:
Hard coded Mempool.Space nodes page on https://mempool.space/lightning/group/mempool.space

<img width="1092" alt="Screen Shot 2022-09-17 at 01 26 59" src="https://user-images.githubusercontent.com/8561090/190830259-9f3dc5fc-23f2-489f-86a3-7c1fc9911d9f.png">

Also a link from the Lightning Dashboard if running official mempool space
<img width="206" alt="Screen Shot 2022-09-17 at 01 26 45" src="https://user-images.githubusercontent.com/8561090/190830292-90165dda-a2d7-47dc-b77f-139ce3920704.png">

wiz commented:
Cool. It would be nice to have a stats table for total channels and liquidity etc. like the ISP page does.

softsimon commented:
Added stats table.
Added channel count and capacity at each node row.

<img width="1070" alt="Screen Shot 2022-09-17 at 11 06 41" src="https://user-images.githubusercontent.com/8561090/190849292-108fc87d-8192-405f-b5f7-8054d72ffd8d.png">

wiz reviewed (approved):
Tested ACK @ v2.5.0-dev [[ad30ba96](https://github.com/mempool/mempool/commit/ad30ba96)]
